docs/setup.md

```diff
@@ -35,10 +34,10 @@ Here are instructions for setting up a Python environment using Miniconda:
 
 Python dependencies are specified using [`poetry`](https://python-poetry.org/).
 
-To install them, open a terminal and enter:
+To install them, open a terminal and ensure you are working in the data-infrastructure repo folder, then enter the following:
```

Comment:
Maybe specify the root folder rather than the repo folder, which to me is a bit ambiguous, like you've done further down

Reply:
That makes sense, I'll update it.